en/developer_guide/multi_device.rst

```diff
@@ -86,3 +86,22 @@ Then, you declare the engine service based on the new class and set the ``templa
                 - @request_stack
                 - %open_orchestra_front.devices%
             alias: templating
+
+Device detection
+----------------
+
+`Wurfl librairy`_ test the User-Agent and normalize devices,
+the server entry point (webserver or reverse-proxy) must add a parameters ``x-ua-device`` to the request header with the device.
```

Comment:
a parameter
i'd rephrase the end : "to the header of the device's request" (if that's what you meant)